tests/integration/test_replicated_database/test.py

```diff
@@ -1327,7 +1333,6 @@ def test_recover_digest_mismatch(started_cluster):
         "sed --follow-symlinks -i 's/Int32/String/' /var/lib/clickhouse/metadata/recover_digest_mismatch/mv1.sql",
         "rm -f /var/lib/clickhouse/metadata/recover_digest_mismatch/d1.sql",
         "rm -rf /var/lib/clickhouse/metadata/recover_digest_mismatch/",  # Will trigger "Directory already exists"
-        "rm -rf /var/lib/clickhouse/store",
```

Comment:
wdyt if we will just run this test the last one in the sequence?

using this fixture `@pytest.mark.run(order=-1)`

Reply:
The fixture doesn't seem to change anything for the flaky test where we run with `./runner ... --random-order`.